datasets/sk/rpvs/crawler.py

```diff
@@ -1,14 +1,183 @@
-        for name in zf.namelist():
-            if not name.endswith(".json"):
-                continue
-            with zf.open(name, "r") as fh:
-                parse_bods_fh(context, fh)
+
+# TODO do we want to add contry "SK" everywhere?
+
+# def crawl(context: Context) -> None:
+#     fn = context.fetch_resource("source.zip", context.data_url)
+#     with zipfile.ZipFile(fn, "r") as zf:
+#         for name in zf.namelist():
+#             if not name.endswith(".json"):
+#                 continue
+#             with zf.open(name, "r") as fh:
+#                 parse_bods_fh(context, fh)
+
+# Exaple URLs
+# https://rpvs.gov.sk/opendatav2/PartneriVerejnehoSektora/44?$expand=Partner,PravnaForma,Adresa
+# https://rpvs.gov.sk/opendatav2/Partneri/20?$expand=Vymaz,Pokuta,OverenieIdentifikacieKUV,konecniUzivateliaVyhod,verejniFunkcionari,kvalifikovanePodnety
+# https://rpvs.gov.sk/opendatav2/KonecniUzivateliaVyhod/45?$expand=Partner,PravnaForma,Adresa
+
+
+def apply_address(context, entity, address):
+    if not address:
+        return
+    street_name = address.pop("MenoUlice")
+    street_number = address.pop("OrientacneCislo", "")
+    city = address.pop("Mesto")
+    city_code = address.pop("MestoKod", "")
+    postal_code = address.pop("Psc")
+
+    address = h.make_address(
+        context,
+        street=f"{street_name} {street_number}".strip(),
+        city=city,
+        place=city_code,
+        postal_code=postal_code,
+        lang="svk",
+    )
+    h.copy_address(entity, address)
+
+
+def fetch_related_data(context, related_id, endpoint):
+    if not related_id:
+        return None
+    related_url = endpoint.format(id=related_id)
+    response = context.fetch_json(related_url, cache_days=3)
+    return response
+
+
+def emit_related_entity(context, entity_data, entity_id, is_pep):
+    first_name = entity_data.pop("Meno")
+    last_name = entity_data.pop("Priezvisko")
+    dob = entity_data.pop("DatumNarodenia")
+
+    # One more flag for public officials (used in ownership relationship)
+    public_official = entity_data.pop("JeVerejnyCinitel")
+
+    if not first_name and not last_name:
+        context.log.warn("No first or last name", entity_data=entity_data)
+        return
+
+    related = context.make("Person")
+    related.id = context.make_id(first_name, last_name, dob)
+    h.apply_name(related, first_name=first_name, last_name=last_name)
+    h.apply_date(related, "birthDate", dob)
+    related.add("title", entity_data.pop("TitulPred"))
+    related.add("title", entity_data.pop("TitulZa"))
+    if address := entity_data.pop("Adresa"):
+        apply_address(context, related, address)
+    if public_official or is_pep:
+        # Based on the internal categorization provided by the source
+        related.add("topics", "role.pep")
+        related.add("country", "SK")
+
+    context.emit(related)
+    context.audit_data(
+        entity_data, ["@odata.context", "Id", "PlatnostOd", "PlatnostDo", "Partner"]
```

Comment:
(nit) If you can, could you add a little comment for the fields what they are and why we're not using them? This might help a future maintainer who's not familiar with the data source.